Payu/Gateways/WC_Gateway_PayuKlarna.php

```diff
@@ -3,21 +3,39 @@
 namespace Payu\PaymentGateway\Gateways;
 
 class WC_Gateway_PayuKlarna extends WC_Payu_Gateways {
-	protected string $paytype = 'dpkl';
+    protected string $paytype = '';
```

Comment:
Do wyrzucenia, w `WC_Payu_Gateways` jest to już w ten sposób zdefiniowane a dodatkowo w konstruktorze to ustawiasz.